openadapt/record.py

```diff
@@ -32,12 +40,15 @@
     "action": True,
     "window": True,
 }
-PLOT_PERFORMANCE = False
-
+PLOT_PERFORMANCE = True
```

Comment:
What do you think about moving this to `config.py`?

Reply:
done in [a9828dd](https://github.com/MLDSAI/OpenAdapt/pull/270/commits/a9828dd9dd5de9e7431e6ba713fa1169e1b8b2e1)